test/pg_handler/etc/cert.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+BASEDIR=$(dirname "$0")
+LOGGING_PREFIX="gen_cert.sh >> "
+
```

Comment:
I wonder if we should be re-using something standard here.  eg, there are:

- https://github.com/conjurdemos/conjur-intro/tree/master/tools/simple-certificates
- https://github.com/conjurinc/appliance/tree/master/features/support/ca
- https://github.com/cyberark/conjur/blob/master/cucumber/api/features/support/ca_helpers.rb

we seem to be duplicating a lot of work around certs as a team...

Reply:
agreed! in fact this whole script is not necessary. we only need the certs. those could be generated once and checked into source control.